### DISCUSSION TITLE
[Suggestion] LinkedIn CSS Blocks plugin

### DISCUSSION THREAD

**Discussion by tarciozemel** [brunch/brunch#1866] [Ideas]
Do you guys saw the [LinkedIn post](https://engineering.linkedin.com/blog/2018/04/css-at-scale--linkedins-new-open-source-projects-take-on-stylesh) announcing their new CSS compiler [CSS Blocks](https://github.com/linkedin/css-blocks)?

Anxious for the Brunch package to test it! :grinning:

**Comment by kgcreative** [comment]
+1
